### PR TITLE
Added traefik docs

### PR DESCRIPTION
**Pull request by TakenMC**

### CODE REVIEW

File: docs/traefik.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+title: traefik reverse proxy
+---
+
+# Traefik static config file
+
+/etc/traefik/traefik.yml
+```yaml
+api:
+  dashboard: true
+  insecure: true
```
Comment (jmattheis) [inline docs/traefik.md:12]: I'm okay with having a traefik guide on the website, but I'd want this to be a generic one, so nothing cloudflare specific. With http acme most of the settings aren't needed as it's the default.

File: docs/traefik.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+entryPoints:
```
Comment (jmattheis) [inline docs/traefik.md:14]: Without the certificateResolvers for cloudflare, this settings file is pretty small and should be configured as traefik cli arguments.

**Review by jmattheis — CHANGES_REQUESTED** [review]
Thanks for the contribution.

File: docs/traefik.md (hunk 2 above)
Comment (TakenMC) [reply docs/traefik.md:14]: ill fix this later

File: docs/traefik.md (hunk 1 above)
Comment (TakenMC) [reply docs/traefik.md:12]: I'll fix this later

**Comment by TakenMC** [comment]
is this better

**Review by jmattheis — APPROVED** [review]
I've simplified it more, no need for different docker compose, and the manual creation of the network.

**Pull request merged**